app/utils/aws.ts

```
@@ -0,0 +1,432 @@
+      if (eventMatch) {
+        try {
+          return JSON.parse(eventMatch[1]);
+        } catch (e) {
+          return { output: eventMatch[1] };
+        }
+      }
+
+      // Handle plain text responses
+      if (text.trim()) {
+        // Clean up any malformed JSON characters
+        const cleanText = text.replace(/[\x00-\x1F\x7F-\x9F]/g, "");
```

Biome (1.9.4)

[error] Unexpected control character in a regular expression.

Control characters are unusual and potentially incorrect inputs, so they are disallowed.

(lint/suspicious/noControlCharactersInRegex)